WebisD/http-api-without-lib
# Feat create PUT method and add some changes to other existing classes

### **Why**
- Created a PUT method necessary whenever the client needs to update existing data.

### **What was done**
- Created proper logic on PUT::response().
- Updated UserObj::class and HandlerDatabase::class with some utility functions and made them a bit more readable.
- Fixed some bugs on list.html and post.html that came from changing the UserObj::class and the HandlerDatabase::class.
- Created put.html which is a copy of post.html and is responsible for the PUT request from the client.

### **How to test**
- Run the `main.py` file.
- Open your browser and go to `localhost:8081/`.
- Try to add a new pokemon, edit and then delete it. 
 
The process should be similar to the one below:

https://user-images.githubusercontent.com/63318342/115973812-374ffe00-a52e-11eb-9ed9-1f16440b006f.mp4
